lib/summarization/strategies/hot_topic_gists.rb

```diff
@@ -13,20 +13,21 @@ def targets_data
 
           op_post_number = 1
 
-          last_twenty_posts =
+          hot_topics_recent_cutoff = Time.zone.now - SiteSetting.hot_topics_recent_days.days
+
+          recent_hot_posts =
```

Comment:
In some corner cases this may return nothing. I think in this case we should pick the last X, like it was done before, but now only when this set is empty.